### ISSUE TITLE
Bug: Enable backports on debian for wireguard

### ISSUE THREAD

**Issue by robjloranger** [VivumLab/VivumLab#129]
**Describe the bug**
Without the PPA mentioned in #128 there is no target package for `wireguard` on Debian without enabling the backports repo.

**To Reproduce**
1. Host with Debian 10
2. Configure
3. Deploy

**Expected behavior**
The task `Install WireGuard` should not fail.

**Desktop (please complete the following information):**
 - OS: Debian 10.6
 - Service: n/a
 - Vivumlab Version: v0.3.0

**Additional context**
Error:
```
TASK [vivumlab_wireguard : Install WireGuard] ************************************************************
fatal: [myserver]: FAILED! => {"changed": false, "msg": "No package matching 'wireguard' is available"}
```

**Comment by voryzen** [comment]
It is not within the scope of VivumLab to initiate, nor delete repositories; in order to do this, VivumLab would require editing a system file, that may possibly break the system. (I acknowledge that VivumLab sets the EPEL repository when installing on a CentOS system, this was discussed, and agreed upon by the core developers of VivumLab)

With the above exception, VivumLab currently installs programs and manipulates those programs; the editting of system files such as this, if incorrect, may break the entire OS. As such, we have purposefully hesitated from editing other files, such as the fstab file.

It states in the VivumLab statement (please read the docs) that VivumLab assumes some knowledge on behalf of the user. Knowing which repos you may need, or how to add them in if it's required, is a part of that assumed knowledge.

tl;dr, this is intentional, and not a bug. you will need this knowledge to successfully run your server anyway, so, enable the repo if you want it. Such intuition may make it into a future release of VivumLab, however at this stage, not a priority

**Comment by voryzen** [comment]
In response to this issue,
at this point in time, backports will only be enabling if the user attempts to run wireguard.

Hope this works for everyone

closing this will close #128

**Comment by robjloranger** [comment]
Yes perfect, that's what I had my local copy doing. 😊

**Comment by voryzen** [comment]
> Yes perfect, that's what I had my local copy doing.

Are yoi referring to the code itself, or the explanation?

**Comment by robjloranger** [comment]
The explanation, I'm not sure about the unstable repo channel. I didn't find it was needed and would have potential to destabilize the system.

**Comment by voryzen** [comment]
> The explanation, I'm not sure about the unstable repo channel. I didn't find it was needed and would have potential to destabilize the system.

So does backports, hence my earlier post.

I'm happy to explain my rationale though.
Debian 9 requires unstable/testing repo.

Ansibke will only run the task to install, if the debian version is not 10 or 11.
This can be rethought whwn debian reaches version 12, but until then, it should be good.

Test away, and tell me what breaks
